subjects/java/checkpoints/singleton-blueprint/ExerciseRunner.java

```diff
@@ -0,0 +1,4 @@
+public class ExerciseRunner {
```

Comment:
Same as in the README file, and its missing the last line

Reply:
done